### PR TITLE
Doc: Particles Init & Manipulators

### PR DESCRIPTION
**Pull request by ax3l**
- document available particle init and manipulators
- document PMacc `struct` factory #1879

### CODE REVIEW

File: docs/source/dev/pmacc.rst [hunk 1]
```diff
@@ -132,3 +132,33 @@ SimulationHelper
    :members:
    :protected-members:
    :undoc-members:
+
+Struct Factory
+--------------
+
+Convinient syntax to generate structs, e.g. in `densityConfig.param`.
```
Comment (PrometheusPi) [inline docs/source/dev/pmacc.rst]: REQUIRED:
Typo: `Convenient`

File: docs/source/dev/pmacc.rst [hunk 2]
```diff
@@ -132,3 +132,33 @@ SimulationHelper
...
+Allows to conviniently switch between variable and constant defined members without the need to declare or initialize them externally.
```
Comment (PrometheusPi) [inline docs/source/dev/pmacc.rst]: REQUIRED:
Typo: `conveniently`

**Comment by ax3l** [comment]
> Have you tested the resulting documentation?

yes, it builds locally.

**Comment by PrometheusPi** [comment]
@ax3l Thanks - great. :+1:

**Pull request merged**